README.md

```diff
@@ -963,7 +963,7 @@ Other Style Guides
     });
 
     // good
-    [1, 2, 3].map(number => `A string containing the ${number}.`);
+    [1, 2, 3].map(number => `A string containing the ${number + 1}.`);
```

Comment:
This remains correct, but why is this “fixing” anything? How does output of 2/3/4 make this example better than it being 1/2/3?

Reply:
The fix is to make the "good example" output the same value as bad example.